test/e2e/network/netpol/probe.go

```diff
@@ -41,7 +41,12 @@ type ProbeJobResults struct {
 
 // ProbePodToPodConnectivity runs a series of probes in kube, and records the results in `testCase.Reachability`
 func ProbePodToPodConnectivity(k8s *kubeManager, model *Model, testCase *TestCase) {
-	numberOfWorkers := 3 // See https://github.com/kubernetes/kubernetes/pull/97690
+	var numberOfWorkers int
+	if framework.NodeOSDistroIs("windows") {
```

Comment:
Handling this in `Model` allows you to centralize the business logic in a clear way that's separate from the infra.